### PR TITLE
Added code to check swss on all asics if dut is_multi_asic

### PR DESCRIPTION
**Pull request by mannytaheri**
### Description of PR
swss docker should be check on all ASICs if  dut is multi ASICs.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
To ensure swss docker is checked on all ASICs if the dut is multi ASICs 

#### How did you do it?
dded code to config_system_checks_passed definition to check all ASICs if dut is multi ASICs.  Otherwise will check one ASIC.
  - If dut is multi_asic
    - duthost.shell("systemctl show swss@{}.service --property ActiveState --value".format(asic.asic_index))
    - duthost.shell("ps -o etimes -p $(systemctl show swss@{}.service --property ExecMainPID --value) | sed '1d'".format(asic.asic_index))

  - If dut is not multi_asic
    - duthost.shell("systemctl show swss.service --property ActiveState --value")
    - duthost.shell("ps -o etimes -p $(systemctl show swss.service --property ExecMainPID --value) | sed '1d'")
    - 
#### How did you verify/test it?
Ran test_reload_configuration and test_reload_configuration_checks testcases which call config_system_checks_passed on a multi ASICss chassis.

### CODE REVIEW

**Comment by rlhui** [comment]
@tjchadaga  - please help prioritize reviewing this, thanks.

**Review by tjchadaga — APPROVED** [review]

File: tests/common/config_reload.py [hunk 1]
```diff
@@ -16,13 +16,24 @@ def config_system_checks_passed(duthost):
...
+        out = duthost.shell("systemctl show swss.service --property ActiveState --value")
+        if out["stdout"] != "active":
+            return False
 
-    out = duthost.shell("ps -o etimes -p $(systemctl show swss.service --property ExecMainPID --value) | sed '1d'")
-    if int(out['stdout'].strip()) < 120:
-        return False
+        out = duthost.shell("ps -o etimes -p $(systemctl show swss.service --property ExecMainPID --value) | sed '1d'")
+        if int(out['stdout'].strip()) < 120:
+            return False
```
Comment (ZhaohuiS) [inline tests/common/config_reload.py:36]: Feel a little curious why is it dark green here? Does indentation use tab or 4 spaces?

Comment (mannytaheri) [reply tests/common/config_reload.py:36]: In the code we have spaces. Using vi, tried changing it to tabs, but git shows the file is the same and won't allow to commit.

I think the code is OK. It is just how the diff is displayed - the original code was left indented by 1 tab - that is possibly why.

**Pull request merged**